runtime/compiler/optimizer/J9ValuePropagation.cpp

```diff
@@ -1508,6 +1508,91 @@ J9::ValuePropagation::constrainRecognizedMethod(TR::Node *node)
       }
    }
 
+TR_YesNoMaybe
+J9::ValuePropagation::isArrayCompTypeValueType(TR::VPConstraint *arrayConstraint)
+   {
+   TR_YesNoMaybe isValueType = TR_maybe;
```

Comment:
My apologies, Annabelle @a7ehuo!  I know you made this same comment in my [earlier OMR pull request #5904](https://github.com/eclipse/omr/pull/5904#discussion_r609952927), which I have since withdrawn, and I had forgotten to address the comment here.

Reply:
No worries. In this review, I realized it'd be easier to return directly instead of saving it to a variable.

Reply:
Switched to using `return` statements in commit cfd28b6087da2a2f87c89c904c1b7d5bc57c6567